sshfs.c

```diff
@@ -46,9 +46,9 @@
 #include <grp.h>
 #include <limits.h>
 #ifdef __APPLE__
-#  include <strings.h>
-#  include <libgen.h>
-#  include <darwin_compat.h>
+#include <strings.h>
+#include <libgen.h>
+#include <darwin_compat.h>
```

Comment:
I don't think there is any issue here, the space is used as an indent to show that these files are included only when `__APPLE__` is defined.

Reply:
Ok i'll modify it.